.zenodo.json

```diff
@@ -223,6 +223,18 @@
       "name": "Ghosh, Satrajit S.",
       "orcid": "0000-0002-5312-6729",
       "type": "Researcher"
+    },
+    {
+      "affiliation": "Max Planck Institute for Human Cognitive and Brain Sciences, Leipzig, Germany",
```

Comment:
Please add your names to the `.maint/contributors.json` file (also, remember names are ordered alphabetically there).

Sorry that the documentation is out of date w.r.t. this too :(

Reply:
We have added our names to the contributors.json file and committed your suggestions for the documentation. 
Do we have to undo our changes in the zenodo file?